lua/wikis/commons/Widget/MainPage/WantToHelp.lua

```diff
@@ -0,0 +1,165 @@
+---@return Widget
+local function buildFontAwesomeList(icon, entries, listCss)
+	return HtmlWidgets.Ul{
+		classes = {'fa-ul'},
+		css = listCss,
+		children = Array.map(entries, function (entry)
+			return HtmlWidgets.Li{
+				children = WidgetUtil.collect(
+					HtmlWidgets.Span{
+						classes = {'fa-li'},
+						children = icon
+					},
+					unpack(entry)
+				)
+			}
+		end)
+	}
+end
+
+---@return string
+local function getWikiType()
+	if Info.wikiName == 'formula1' then
+		return 'F1'
+	end
+	return 'esports'
+end
+
+---@param content Widget?
+---@return Widget
+local function showWhenLoggedOut(content)
+	return Div{
+		classes = {'show-when-logged-out'},
+		children = content
+	}
+end
+
+---@return Widget[]
+function WantToHelp:render()
+	return {
+		Div{
+			css = {['padding-bottom'] = '0.7em'},
+			children = 'Create your free account and join the community to start making a difference by ' ..
+					'sharing your knowledge and insights with fellow ' .. getWikiType() ..
+					' fans!'
+		},
+		buildFontAwesomeList(
+			GREEN_CHECK_CIRCLE,
+			{
+				{'Join our community and grow the scene(s) you care about.'},
+				{'Be a hero for fans worldwide by keeping the site updated.'},
+				{'Develop valuable skills in research, writing, and collaboration.'},
+			},
+			{
+				margin = '0 0 0 2.5rem',
+				['line-height'] = '2.2em'
+			}
+		),
+		Div{
+			css = {
+				display = 'flex',
+				['flex-wrap'] = 'wrap',
+				gap = '12px',
+				['justify-content'] = 'center'
+			},
+			children = WidgetUtil.collect(
+				showWhenLoggedOut(Button{
+					link = 'https://tl.net/mytlnet/register?utm_source=Liquipedia&utm_medium=Website' ..
+						'&utm_campaign=Want+to+Help+' .. mw.uri.encode(mw.site.siteName) .. '&utm_id=Want+to+Help',
+					linktype = 'external',
+					title = 'Click here to create an account',
+					children = {
+						IconFa{iconName = 'createaccount'},
+						' Create Account'
+					}
+				}),
+				showWhenLoggedOut(Button{
+					link = 'Special:UserLogin',
+					title = 'Click here to log in',
+					variant = 'secondary',
+					children = {
+						IconFa{iconName = 'login'},
+						' Log In'
+					}
+				}),
+				Button{
+					link = 'https://discord.gg/liquipedia',
+					linktype = 'external',
+					title = 'Click here to join our discord server',
+					variant = 'secondary',
+					children = {
+						IconFa{iconName = 'discord'},
+						' Join Our Discord'
+					}
+				},
+				Page.exists('Help:Contents') and Button{
+					link = 'Help:Contents',
+					title = 'Click Here to Read our Help Articles',
+					variant = 'secondary',
+					children = {
+						IconFa{iconName = 'helparticles'},
+						' Help Articles'
+					}
+				} or nil
+			)
+		},
+		HtmlWidgets.Hr{
+			css = {['margin-top'] = '1em'}
+		},
+		Div{
+			css = {
+				['line-height'] = '2em',
+				['margin-top'] = '1em'
+			},
+			children = {'\n', WantToHelpList.get{}}
+		},
+		HtmlWidgets.Br{},
+		'In total there are ',
+		Link{
```

Comment:
iirc not all wikis have the `Liquipedia:Want to help/All` page
question is if we want to enforce it everywhere or if we want to not link if the page doesn't exist